client/protractor-conf.js

```diff
@@ -25,7 +25,7 @@ exports.config = {
     framework: 'jasmine2',
     jasmineNodeOpts: {
         showColors: true,
-        defaultTimeoutInterval: 55000
+        defaultTimeoutInterval: 120000
```

Comment:
Anyway seams that larger tests are not a good idea, so we have to talk about it on retrospective meeting.